src/main/jbake/content/blog/2017/02/04/hawkular-apm-service-deployments.adoc

```diff
@@ -0,0 +1,38 @@
+= Hawkular APM: Comparing performance of service versions
+Gary Brown
+2017-02-04
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, openshift
+
+With the release of https://github.com/hawkular/hawkular-apm/releases/tag/0.14.0.Final[0.14.0.Final on Hawkular APM] we have introduced a new feature in support of continuous deployment of microservices.
+When a new version of a service is deployed, potentially as part of a deployment strategy (such as Canary, A/B, Blue/Green), it is important
+to be able to assess the impact the new version of the service has in comparison to existing/previous versions.
+
+The new "Services" tab in the UI provides just such a feature. When trace data is reported by the Hawkular APM OpenTracing provider (Java
+or JavaScript), it identifies the service name and optional version (known as a buildStamp). When the services are deployed within OpenShift
```

Comment:
I would not put the "When trace data ... " part here, as everything else is about the UI. This how-to-identify is more like a footnote to me.
Why not put that in a section after the chart?